src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,150 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const PeopleFilters = () => {
+  const centuriesList = [16, 17, 18, 19, 20];
```

Comment:
I suggest writing it before you write functional component and in all capitals as it is a constant:
const CENTURIES_LIST = [16, 17, 18, 19, 20];